### PR TITLE
Only display 4 PR

### PR DESCRIPTION
**Pull request by zuik**
Addressed #8 
- Also ran the file through a prettifier
- Also get rid of the comma underneath each PR

### CODE REVIEW

**Comment by adeora7** [comment]
Can you please resolve the merge conflict. Sorry for late reply.

File: popup.js [hunk 1]
```diff
@@ -40,31 +37,46 @@ function getMessage(total_count){
...
-      var count = (data['total_count']>4?"4":data['total_count'])+ "/4";
-      res +="<div id='prCompleteCount'>"+count+"</div>";
+      res += "<div id='resultHandle'>" + handle + "</div>";
+      var count = (data['total_count'] > 4
+        ? "4"
+        : data['total_count']) + "/4";
+      res += "<div id='prCompleteCount'>" + count + "</div>";
       var message = getMessage(data['total_count']);
-      res +="<div id='message'>"+message+"</div>";
-	  if (data['items'].length > 0) {
```
Comment (adeora7) [inline popup.js]: This condition should be there but is missing from your code. It is to show "Pull requests" only if number of PRs is more than 0. @zuik

Comment (zuik) [reply popup.js]: I added it back in.

**Comment by adeora7** [comment]
Hey, this issue has already been resolved.

**Pull request closed without merging**